### PR TITLE
add has-payout-account, is-refunded

### PR DESCRIPTION
**Pull request by humbamp123**

### CODE REVIEW

**Comment by humbamp123** [comment]
Just to be clear, teller shouldn't be in blueprints at all?

**Comment by jalehman** [comment]
That’s correct. Anything that needs both should be in Odin at this point, and if it’s used in multiple places it would be in a utility namespace.

**Comment by humbamp123** [comment]
Have yet to test if this works. @allenjprice

**Comment by allenjprice** [comment]
try pulling from `development` and running this code now

File: src/blueprints/models/account.clj [hunk 1]
```diff
@@ -1,10 +1,10 @@
 (ns blueprints.models.account
   (:require [blueprints.models.application :as application]
-            [blueprints.models.approval :as approval]
             [clojure.spec.alpha :as s]
+            [clojure.string :as string]
             [datomic.api :as d]
             [toolbelt.datomic :as td]
-            [clojure.string :as string]))
+            [taoensso.timbre :as timbre]))
```
Comment (allenjprice) [inline src/blueprints/models/account.clj]: Clean up logs, plz

**Pull request merged**